index.html

```diff
@@ -56,7 +52,161 @@ <h4>I’am a Software Developer</h4>
+                        <span class="tag">bootstrap</span>
+                        <span class="tag">Ruby</span>
+                    </div>
+                    <div class="card-button">
+                        <button class="button button-display">See Project</button>
+                    </div>
+                </div>
+                <div class="card4">
+                    <div class="card-content">
+                        <h5>Profesional Art Printing Data More</h5>
+                        <p class="content-paragraph">A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been
+                            the industry's standard</p>
+                    </div>
+                    <div class="card-info">
+                        <span class="tag">html</span>
+                        <span class="tag">bootstrap</span>
+                        <span class="tag">Ruby</span>
+                    </div>
+                    <div class="card-button">
+                        <button class="button button-display">See Project</button>
+                    </div>
+                </div>
+                <div class="card5">
+                    <div class="card-content">
+                        <h5>Data Dashboard Healthcare</h5>
+                        <p class="content-paragraph">A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been
+                            the industry's standard</p>
+                    </div>
+                    <div class="card-info">
+                        <span class="tag">html</span>
+                        <span class="tag">bootstrap</span>
+                        <span class="tag">Ruby</span>
+                    </div>
+                    <div class="card-button">
+                        <button class="button button-display">See Project</button>
+                    </div>
+                </div>
+                <div class="card6">
+                    <div class="card-content">
+                        <h5>Website Portfolio</h5>
+                        <p class="content-paragraph">A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been
+                            the industry's standard</p>
+                    </div>
+                    <div class="card-info">
+                        <span class="tag">html</span>
+                        <span class="tag">bootstrap</span>
+                        <span class="tag">Ruby</span>
+                    </div>
+                    <div class="card-button">
+                        <button class="button button-display">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about-me">
+        <div class="about-info">
+            <h1>About me</h1>
```

Comment:
- [ ] Please, consider that you could have one `h1` on your webpage. All other headers should have `hx` tag (aka h1, h2, h3, etc..) - try to replace it everywhere in your code.